src/ResourceManager/Network/Commands.Network/packages.config

```diff
@@ -1,6 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="AutoMapper" version="6.2.2" targetFramework="net452" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Network" version="19.7.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="19.6.2-preview" targetFramework="net452" />
```

Comment:
@akshaysngupta same comment about downgrading the Network SDK version

Reply:
removed.